docs/02-app/01-building-your-application/08-testing/02-jest.mdx

```diff
@@ -88,14 +90,14 @@ module.exports = createJestConfig(config)
 
 Under the hood, `next/jest` is automatically configuring Jest for you, including:
 
-- Setting up `transform` using the [Next.js Compiler](/docs/architecture/nextjs-compiler)
-- Auto mocking stylesheets (`.css`, `.module.css`, and their scss variants), image imports and [`next/font`](/docs/pages/building-your-application/optimizing/fonts)
-- Loading `.env` (and all variants) into `process.env`
-- Ignoring `node_modules` from test resolving and transforms
-- Ignoring `.next` from test resolving
-- Loading `next.config.js` for flags that enable SWC transforms
+- Setting up `transform` using the [Next.js Compiler](/docs/architecture/nextjs-compiler).
+- Auto mocking stylesheets (`.css`, `.module.css`, and their scss variants), image imports and <PagesOnly>[`next/font`](/docs/pages/building-your-application/optimizing/fonts)</PagesOnly> <AppOnly>[`next/font`](/docs/app/building-your-application/optimizing/fonts)</AppOnly>.
```

Comment:
The separate links like this shouldn't be necessary we can leave the link just to the app docs instead since it's the same content

Reply:
Sure! I've fixed it.

> The separate links like this shouldn't be necessary we can leave the link just to the app docs instead since it's the same content

Sure! I've fixed it and made it use `app` instead of `pages`.